### PR TITLE
remove "field" from group-by dropdown and add tooltips to "form: fiel…

### LINKED ISSUE #16: Remove "Field" from group by control
Just use form field. Will also need to update clickable y-axis a bit.

### PR DESCRIPTION
**Pull request by samussiah**
…d" y-axis tick labels

Closes #16

### CODE REVIEW

File: src/chart/onResize.js [hunk 1]
```diff
@@ -164,15 +173,17 @@ export default function onResize() {
             this.config.formDescription_col
           ]
       );
-  if (this.config.y.column === "Field" && this.config.fieldDescription_col)
+  if (this.config.y.column === "Form: Field" && this.config.fieldDescription_col)
```
Comment (jwildfire) [inline src/chart/onResize.js]: This isn't giving a title for me. Also, it should probably be: `this.config.y.column === "Form: Field" && (this.config.fieldDescription_col || this.config.formDescription_col)`

Comment (jwildfire) [reply src/chart/onResize.js]: Actually - not getting titles in `v1.0.0-dev` branch either, so probably an issue I introduced with the merge.

File: src/chart/onResize.js [hunk 2]
```diff
@@ -164,15 +173,17 @@ export default function onResize() {
...
+      .style('cursor', 'help')
       .filter(field => this.y_dom.indexOf(field) > -1)
       .append("title")
       .text(
-        field =>
-          this.raw_data.filter(d => d.Field === field)[0][
-            this.config.fieldDescription_col
-          ]
-      );
+        field => {
```
Comment (jwildfire) [inline src/chart/onResize.js:181]: how about something like this? 
```
myFormDescription = this.config.formDescription_col ? datum[this.config.formDescription_col] : "<<none>>"
myFieldDescription = <same idea as above>;
return "Form: "+myFormDescription+"\nField:"+myFieldDescription;
```

**Review by jwildfire — CHANGES_REQUESTED** [review]
Not seeing titles in CAT. Also suggested a few minor coding tweaks.

**Comment by samussiah** [comment]
Branch updated.

**Review by jwildfire — CHANGES_REQUESTED** [review]
I'm still not actually getting any titles when I mouseover the y-axis :(

**Comment by samussiah** [comment]
The CDN cache hadn't updated.  Should be good to go now @jwildfire .

**Review by jwildfire — APPROVED** [review]

**Comment by jwildfire** [comment]
Looks good - merging.

**Pull request merged**